tests/device_tests/test_msg_webauthn.py

```diff
@@ -0,0 +1,203 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2019 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from trezorlib import messages as proto
+from trezorlib.exceptions import Cancelled, TrezorFailure
+
+from ..common import MNEMONIC12
+
+CRED1 = bytes.fromhex(
+    "f1d00200f8221312f7898e31ea5ec30409527c2b0bde0b9dfdd7eaab4424173f"
+    "bf75ab67627fff60974460d903d7d96bb9e974c169a01b2c38cf2305da304169"
+    "d4e28f59053a2564bebb3eb3f06c2182f1ea4a2f7cebd8f92a930a76f3b45334"
+    "1e3f3285a575a54bcba9cf8a088dbfe24e8e691a5926160174e03aa941828f49"
+    "e42b47804d"
+)
+
+CRED2 = bytes.fromhex(
+    "f1d00200eb3b566f4ea0a219552b2efd2c76e1ffc2e641d3bf91ec92d47a4ed4"
+    "d78cf42845248c4e982a503618bac0cecfb0fa91fa10821df1efe1d59ac8314e"
+    "b57eb7f32a1a605f91e8692daf1a679b55ab1acadfded5e0c7fd1365e2801759"
+    "bd3a4450dd5589586ab072da79"
+)
+
+CRED3 = bytes.fromhex(
+    "f1d00200ebee50034eb7affb555602eed0812b63d158b57a4188523ad064a719"
+    "febf477c52cfcc7ded8d7a7a83af52287ed1ecee9f74f62b7e55ad8e814c062e"
+    "009bb3b3391dfec79dc93053b0279eca7207358a0962865da55668b2509de773"
+    "8c819dbeead9997778319ac1f1c7318fd6"
+)
+
+CREDS = [
```

Comment:
this doesn't seem to be used for anything, except for one `CREDS[14]` reference?

Reply:
The first 14 are used on line 177 and the last one is used on line 198 below.